templates/Pages/rules_submission.php

```diff
@@ -110,6 +110,16 @@
             "<span class="code">beegfs_ctl</span>"; however, additional
             subdirectories within these directories cannot be precreated.
         </li>
+        <li>
+            Submissions received after the posted deadline for a list may
```

Comment:
How about, "may be accepted (at the discussion of the io500 committee) up request to..."